src/main/java/org/openhmis/dto/AccountDTO.java

```diff
@@ -0,0 +1,49 @@
+
+
+package org.openhmis.dto;
+
+
+import java.util.Date;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import org.openhmis.code.None;
+import org.openhmis.code.YesNo;
+import org.openhmis.code.YesNoReason;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+import org.openhmis.dto.UserDTO;
+
+
+@XmlRootElement
+public class AccountDTO extends BaseDTO {
+    private String externalId;
+    private UserDTO user;
+
+    public AccountDTO() {}
```

Comment:
This class needs documentation.  (I realize that the existing DTOs don't provide a great example of documentation, but never a time like the present for raising our standards, ahem :-) ).